### DISCUSSION TITLE
杜比大喇叭模块配合网易云音乐，制作完安装后APP无法打开

### DISCUSSION THREAD

**Discussion by SekiBetu** [LSPosed/LSPatch#28] [Q&A]
https://music.163.com/#/download
https://github.com/nining377/dolby_beta/releases/tag/v3.3.3
![image](https://user-images.githubusercontent.com/38283893/139028877-214e45b8-e029-4150-867e-bdf042d3fadc.png)
![image](https://user-images.githubusercontent.com/38283893/139028900-03a715bc-5a83-4614-8338-225cbf698b16.png)

**Comment by SuperbutterX** [comment]
杜比大喇叭作者说了网易云7.3.30版本之后的安卓12闪退
